src/ml/ml.cc

```diff
@@ -83,7 +72,7 @@ ml_dimension_calculated_numbers(ml_worker_t *worker, ml_dimension_t *dim, const
 
     storage_engine_query_init(dim->rd->tiers[0].seb, dim->rd->tiers[0].smh, &handle,
               training_response.query_after_t, training_response.query_before_t,
-              STORAGE_PRIORITY_BEST_EFFORT);
+              STORAGE_PRIORITY_SYNCHRONOUS);
```

Comment:
@ktsaou Revert this change just so that we can tell what are the differences in the ML behaviour that are caused only by the scheduling changes. Once we do that we can re-enable sync queries.